content/en/blog/grpc-csharp-future.md

```diff
@@ -1,193 +1,82 @@
-  community.
-
-
-### The plan
-
-**Phase 1: Grpc.Core becomes "Maintenance Only"**
+We, the gRPC team, are announcing today a **[phase-out plan][] for the original
+gRPC C# implementation** ([Grpc.Core][]). This post explains why we're doing so,
+provides phase-out plan details, and attempts to answer (anticipated) frequently
+asked questions.
+
+## Evolution of gRPC support for C#
+
+We first introduced C# support for gRPC back in 2016, with a [gRPC-Core based
+implementation][grpc-core-csharp], most often referred to as **[Grpc.Core][]**,
+the name of its nuget package.
+
+In September 2019, we [announced the general availability][ga] of
+**[grpc-dotnet][]**, a new **pure-C#** implementation of gRPC for .NET Core.
+
+{{% alert title="Note" color="info" %}}
+  For a detailed comparison of these two implementations, see [grpc-dotnet and
+  Grpc.Core comparison][comparison].
+
+  [comparison]: https://github.com/grpc/grpc-dotnet/blob/master/doc/implementation_comparison.md
+{{% /alert %}}
+
+Since then, grpc-dotnet has continued to mature and significantly increase
+both its feature set and user-base.
+
+We've reached a point where we can no longer justify the maintenance costs of
+two implementations of gRPC for C#. The Grpc.Core implementation is based on
+design tradeoffs that made sense in 2016, but are no longer optimal -- this
+makes it a good candidate for a progressive retirement.
+
+In contrast, the modern design of grpc-dotnet is the best choice for the
+community that it serves, given its seamless integration with .Net: as a pure C#
+implementation, it will be easier to use and debug. This also makes it more
+contribution-friendly for C# community members.
+
+## Phase-out plan {#plan}
+
+### Phase 1: maintenance-only
 
 **When: Effective immediately (May 2021)**
 
-From now on, we are no longer going to provide new features or enhancements for
-Grpc.Core. Important bugs and security issues will continue to be addressed in a
-normal way.
+Effective immediately, Grpc.Core is in maintenance-only mode: we won't provide
+Grpc.Core with new features or enhancements. Important bugs and security issues
+will continue to be addressed in a normal way.
 
 We will publish Grpc.Core releases normally, with the usual 6 weekly cadence.
 
 The releases will be based on the newest grpc C core native library build, so
 all new features that don't require C# specific work will also be included.
 
-**Phase 2: Grpc.Core becomes "Deprecated"**
+### Phase 2: deprecation
 
 **When: 1 year from now (May 2022)**
 
-Once this milestone is reached, Grpc.Core will no longer be officially supported
-and all the users will be strongly advised to only use grpc-dotnet from this
-point onwards.
-
-The Grpc.Core nuget packages will remain available in the nuget.org repository,
-but no more fixes will be provided (= not even security fixes).
-
-**Future of Grpc.Tools and Grpc.Core.Api nuget packages**
```

Comment:
I'm not sure summarizing the situation around Grpc.Tools and Grpc.Core.Api  in the Q/A is sufficient.
(especially since Grpc.Core.Api contains  "Grpc.Core" in its name, it might make sense to at least put a note in the text that says these packages are going to be handled differently than Grpc.Core (and point folks to more details in Q/A).